src/domain/task/utils/task.utils.ts

```diff
@@ -0,0 +1,17 @@
+import type { DeepReadonly } from 'superTypes'
+
+type Progress = { min?: number; max?: number; current?: number }
+
+export const progressInvariant = (progress: DeepReadonly<Progress>): boolean => {
```

Comment:
I would have called it: `isProgressValid`.

Reply:
Yep, much better